src/main/kotlin/no/nav/tilleggsstonader/sak/migrering/arena/ArenaStatusService.kt

```diff
@@ -0,0 +1,57 @@
+package no.nav.tilleggsstonader.sak.migrering.arena
+
+import no.nav.tilleggsstonader.kontrakter.felles.IdentStønadstype
+import no.nav.tilleggsstonader.kontrakter.felles.Stønadstype
+import no.nav.tilleggsstonader.sak.behandling.BehandlingService
+import no.nav.tilleggsstonader.sak.fagsak.FagsakService
+import no.nav.tilleggsstonader.sak.migrering.routing.SøknadRoutingService
+import no.nav.tilleggsstonader.sak.opplysninger.pdl.PersonService
+import no.nav.tilleggsstonader.sak.opplysninger.pdl.dto.identer
+import org.slf4j.LoggerFactory
+import org.springframework.stereotype.Service
+
+@Service
+class ArenaStatusService(
+    private val personService: PersonService,
+    private val fagsakService: FagsakService,
+    private val behandlingService: BehandlingService,
+    private val søknadRoutingService: SøknadRoutingService,
+) {
+
+    private val logger = LoggerFactory.getLogger(javaClass)
+
+    fun finnStatus(request: ArenaFinnesPersonRequest): ArenaFinnesPersonResponse {
+        return ArenaFinnesPersonResponse(request.ident, finnesPerson(request))
+    }
+
+    private fun finnesPerson(request: ArenaFinnesPersonRequest): Boolean {
+        val identer = personService.hentPersonIdenter(request.ident).identer().toSet()
+        if (finnBehandlingStatus(identer, request.stønadstype)) {
+            logger.info("Sjekker om person finnes i ny løsning finnes=true harBehandling")
+            return true
+        }
+        if (harRouting(identer, request.stønadstype)) {
+            logger.info("Sjekker om person finnes i ny løsning finnes=true harRouting")
+            return true
+        }
+        return false
+    }
+
+    private fun finnBehandlingStatus(
```

Comment:
Synes if-sjekken i finnPerson blir tydligere med dette navnet 🤓 

```suggestion
    private fun harBehandling(
```

Reply:
Enig, takk! Hadde en annen løsning førs tog metodenavnet ble gjenbrukt men hva den returnerte ble annet 😁 Har oppdatert